src/referrals/referrals.service.ts

```
@@ -457,7 +457,7 @@ export class ReferralsService {
   async getReferralReport(dto: ReferralReportRequestDto) {
     const { limit = 10, offset = 0, filters = {} } = dto;
 
-    const { fromSql, whereClause, params } = this.buildReportBase(filters);
+    const { fromSql, whereClause, params } = this.buildReportBase(filters, true);
```

ESLint

[error] Replace `filters,·true` with `⏎······filters,⏎······true⏎····`

(prettier/prettier)